client/api.go

```diff
@@ -148,6 +149,8 @@ type API struct {
 	PaymentSource *paymentsource.Client
 	// ExchangeRates is the client used to invoke /exchange_rates APIs.
 	ExchangeRates *exchangerate.Client
+	// PaymentIntents is the client used to invoke /exchange_rates APIs.
```

Comment:
Slight fix needed in the `/exchange_rates` part of the comment.

Reply:
BTW, we should try to remember to alphabetize both these lists later :)

Reply:
yep I can take after this PR, I did not want to mix both